### PR TITLE
Show approved volunteers

### PR DESCRIPTION
**Pull request by 7alip**
> Anyone can fill the form to be registered as volunteer. In this case their username will be an arbitrary number.
> We should evaluate the applications and approve after updating usernames. We may also create `authenticated user` for some of the volunteers and attach to their profile.

- [x] Fixes: Non-approved volunteers are being listed
- [x] Fixes: Volunteers list is not being updated (now fetches the data in the frontend as well)

### CODE REVIEW

File: src/pages/volunteers.js [hunk 1]
```diff
@@ -95,8 +99,8 @@ export const getStaticProps = async context => {
   return {
     props: {
       ...(await serverSideTranslations(context.locale, ['common'])),
-      volunteers,
-      jobs,
+      volunteers: volunteersResponse.result,
+      jobs: jobsResponse.result,
```
Comment (MustafaTRKYLMZ) [inline src/pages/volunteers.js:103]: Can we add validate here? Is it nessesary?

**Review by MustafaTRKYLMZ — APPROVED** [review]
After the we add validate, I think it is ok.

File: src/pages/volunteers.js (hunk 1 above)
Comment (7alip) [reply src/pages/volunteers.js:103]: What do you mean by validate?

Comment (MustafaTRKYLMZ) [reply src/pages/volunteers.js:103]: For example when we approved some volunteer we will see immedately in the web site. It is like blog page (revalidate:120) It is for refresh website, when we have new volunteer.

Comment (7alip) [reply src/pages/volunteers.js:103]: It's true. But in this update we already fetch the data in the frontend as well.

https://github.com/wsvvrijheid/wsvvrijheid/blob/5c681aa94273c874d91ca8bda505fa59fa5ee6f5/src/pages/volunteers.js#L17

We will handle this synchronisation with #65 later. I'm considering to use [initial data](https://react-query.tanstack.com/guides/ssr#using-initialdata) approach with ISR (`revalidate`)

Comment (7alip) [reply src/pages/volunteers.js:103]: Basicly, if the page is not supposed to be updated immediately after data changes, revalidate is already fixing this issue. 

But be best solution seems to be:

1. `getServerSideProps` which shows always the current data OR
2. `getStaticProps` + `revalidate (ISR)` + `useQuery (frontend fetch)`

**Pull request merged**